utils/file_manager.py

```
@@ -0,0 +1,131 @@
+    
+    def system_search(self):
+        """
+        Search the entire file system for the config file (more thorough but slower)
+        """
+        filename = self.fileName
+
+        print(f"\nPerforming system-wide search for {filename}...")
+        print("⚠️  This may take a while...")
+        print("-" * 50)
+        
+        found_files = ""
```

Ruff (0.11.9)

Local variable `found_files` is assigned to but never used

Remove assignment to unused variable `found_files`

(F841)